pkg/account/storage/v2/account.go

```diff
@@ -374,6 +374,59 @@ func (s *accountStorage) GetAccountV2(ctx context.Context, email, organizationID
 	return &domain.AccountV2{AccountV2: &account}, nil
 }
 
+func (s *accountStorage) GetAccountV2ByEnvironmentID(
+	ctx context.Context,
+	email, environmentID string,
+) (*domain.AccountV2, error) {
+	account := proto.AccountV2{}
+	var organizationRole int32
+	query := `
+		SELECT
```

Comment:
For new queries, can you use an SQL file?

https://github.com/bucketeer-io/bucketeer/tree/main/pkg/feature/storage/v2/sql/flag_trigger

You can read it using `go:embed`.
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/feature/storage/v2/flag_trigger.go#L30-L43

Reply:
@cre8ivejp 
Thank you, I fixed it 👍